build/compile/LzmaArchive.targets

```diff
@@ -17,5 +18,6 @@
                     ContinueOnError="WarnAndContinue" />
 
       <Copy SourceFiles="$(IntermediateArchive)" DestinationFiles="$(FinalArchive)" />
+      <Copy SourceFiles="$(IntermediateArchive)" DestinationFiles="$(PackagesArchive)" />
```

Comment:
This looks like it will only copy the lzma if it is included in the SDK zip/installer. Is that what '$(IncludeNuGetPackageArchive)' == 'true' does? If so, then I think this won't address the request, which was to have flavors (ARM) where the lzma is not included in the zip/installer, but was easily acquired separately.

Reply:
@nguerrera 
I feel this would result in a 'double' publish [Linux-arm + Linux-arm64] which would fail one of the publishes as the name collides: "nuGetPackagesArchive.lzma"

Since the LZMA is not platform specific we are picking a known VSO build to do the singular publish -> Windows x64

Reply:
Oh, I see that we'd only publish it with x64. Is the expectation that users get it from the x64 directory. I guess there's nothing platform specific about it. It will just be confusing in docker files to see ti get downloaded from x64.

Reply:
Oh it's all in one folder. I see. OK.

Reply:
Nevermind this comment thread, then.

Reply:
@nguerrera 
The path to pick this up would be: "http://dotnetcli.blob.core.windows.net/dotnet/Sdk/2.2.100-preview1-008982/nuGetPackagesArchive.lzma"

Reply:
Looks great.